repo/db/bots.go

```diff
@@ -14,19 +14,19 @@ type BotDB struct {
 	modelStore
 }
 
-func NewBotStore(db *sql.DB, lock *sync.Mutex) repo.BotStore {
+func NewBotstore(db *sql.DB, lock *sync.Mutex) repo.Botstore {
 	return &BotDB{modelStore{db, lock}}
 }
 
-// AddOrUpdate Bot KV store adds namespace all bot requests by their id
-func (c *BotDB) AddOrUpdate(id string, key string, value []byte, botVersion int) error {
+// AddOrUpdate Bot KV store adds namespace all bot requests by their key
+func (c *BotDB) AddOrUpdate(key string, value []byte) error {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 	tx, err := c.db.Begin()
 	if err != nil {
 		return err
 	}
-	stm := `insert or replace into botstore(id, key, value, version, created, updated) values(?,?,?,?,coalesce((select created from botstore where id=? and key=?),?),?)`
+	stm := `insert or replace into bots_store(id, value, created, updated) values(?,?,coalesce((select created from bots_store where id=?),?),?)`
```

Comment:
taking the opportunity (of needing to remove the key column) to name it in the same style as other tables